docs/upgrading_to_v1.0.md

```diff
@@ -0,0 +1,35 @@
+# Upgrading to v1.0
```

Comment:
Looks like the next auto-release will be `v0.4.0`, or do you want to increment to `v1`?  If the later, also remove this line: https://github.com/GoogleCloudPlatform/terraform-google-cloud-spanner/blob/main/.github/release-please.yml#L18

Reply:
If it's a breaking change then I would expect major version change, right?

Reply:
Done

Reply:
Optional if it's pre-v1.0

Reply:
ok, I have removed bumpMinorPreMajor